AutoRest/Generators/Ruby/Ruby/TemplateModels/ModelTemplateModel.cs

```diff
@@ -124,6 +124,22 @@ public ModelTemplateModel(CompositeType source, ISet<CompositeType> allTypes)
             PropertyTemplateModels = new List<PropertyTemplateModel>();
             source.Properties.ForEach(p => PropertyTemplateModels.Add(new PropertyTemplateModel(p)));
 
+            if (!string.IsNullOrEmpty(source.PolymorphicDiscriminator))
```

Comment:
Not really. This place we put polymorphic discriminator property (just one property that allows us to distinguish between inheritance) along with other model properties to make sure the mapper includes that property. 

I think the point we were discussing was putting model properties of parent model in mapper of the child model through [ComposedProperties](https://github.com/Azure/autorest/pull/1106/files#diff-ed067c614889b2463c524602fdbda795R629). Isn't it?